src/thor/isochrone.cc

```diff
@@ -217,17 +226,17 @@ void Isochrone::UpdateIsoTile(const EdgeLabel& pred,
     auto tile1 = isotile_->TileId(ll0);
     auto tile2 = isotile_->TileId(ll);
     if (tile1 == tile2) {
-      isotile_->SetIfLessThan(tile1, secs1 * kMinPerSec);
+      SetData(tile1, {secs1 * kMinPerSec, dist1 * kMetersPerKm});
```

Comment:
this new method will update the grid's time and/or distance value depending if it's smaller than the current value(s).